kafka/pom.xml

```diff
@@ -198,7 +198,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_2.11</artifactId>
-      <version>0.9.0.0</version>
```

Comment:
It's a backward compatible patch change, I don't see why it would cause a problem. On the contrary there was earlier some code trying to use the new operator that had to redefine the dependency to 0.9.0.1 due to a bug.